### ISSUE TITLE
Stutters

### ISSUE THREAD

**Issue by awwe** [sigmaxipi/chromium-for-stadia#31]
Hello. I have frequent stutter when playing this on my quest 2 now on 90hz. 

Why? Is it because of 90hz?

I have smooth gameplay with stadia on the same 5ghz network on Otter devices.

**Comment by sigmaxipi** [comment]
Yes. Stadia runs at 60Hz. You'll need to run `adb shell setprop debug.oculus.refreshRate 60` to force your headset into 60Hz mode for best performance. You can see the current FPS by running `adb logcat *:S VrApi:*`
